Assets/MRTK/SDK/Features/Utilities/UnitTests/LRUCacheUnitTests.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MRTK.SDK.Utilities.UnitTests
```

Comment:
Our pattern for v2 is to put tests in https://github.com/microsoft/MixedRealityToolkit-Unity/tree/main/Assets/MRTK/Tests. Can this please be moved there in the corresponding folder (edit mode vs play mode, etc)? This namespace will also need to be updated to match the v2 conventions.

Reply:
I've moved this to `Assets/MRTK/Tests/PlayModeTests/Core/Utilities/LRUCacheUnitTests`